### PR TITLE
update create first app page

### PR DESCRIPTION
**Pull request by PetrakovichVictoria**

### CODE REVIEW

File: topics/setup.md [hunk 1]
```diff
@@ -6,13 +6,13 @@ Before you begin [creating your first application](create-first-app.md) to work
 1. If you are going to work with shared code or Android-specific code, you can work on any computer with an operating 
    system supported by [Android Studio](https://developer.android.com/studio).  
    If you also want to write iOS-specific code and run an iOS application on a simulated or real device, use a Mac with a 
-   macOS.  
+   macOS. These steps cannot be performed on other operating systems, such as Microsoft Windows. This is due to an Apple requirement.
 2. Install [Android Studio](https://developer.android.com/studio) – version 4.1 or higher.
```
Comment (PetrakovichVictoria) [inline topics/setup.md:10]: Should I leave 4.1 here because 4.2 is only Beta and Canary yet? (If yes, then I should also update versions of Xcode and Kotlin plugin.

File: topics/setup.md [hunk 2]
```diff
@@ -6,13 +6,13 @@ Before you begin [creating your first application](create-first-app.md) to work
...
 2. Install [Android Studio](https://developer.android.com/studio) – version 4.1 or higher.   
     You will use Android Studio for creating your multiplatform applications and running them on simulated or hardware devices.
-3. If you need to write iOS-specific code and run an iOS application, install [Xcode](https://apps.apple.com/us/app/xcode/id497799835) 
+3. If you need to write iOS-specific code and run an iOS application, install [Xcode](https://apps.apple.com/us/app/xcode/id497799835)
     –  version 11.3 or higher.                                                                                                                                                                                                                                                                                                                          
     Most of the time, Xcode will work in the background. You will use it to add Swift or Objective-C code to your iOS application.
-4. Update the Kotlin plugin to version 1.4.20 or higher.  
+4. If you have used your current installation of Android Studio before, update the Kotlin plugin to version 1.4.20 or higher.
```
Comment (PetrakovichVictoria) [inline topics/setup.md:15]: Because if not, then you don't need to do anything. If you have just installed AS – everything is already up to date.

File: topics/setup.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ Before you begin [creating your first application](create-first-app.md) to work
     
     Check out [KMM plugin release notes](kmm-plugin-releases.md).
     
-6. Install the [JDK](https://jdk.java.net/14/) if you haven't already done so.
```
Comment (PetrakovichVictoria) [inline topics/setup.md:26]: By the old link it was said that "JDK 14 has been superseded. Please visit jdk.java.net for the current version."
So I would offer to point a user to the most fresh versions.

File: topics/setup.md (hunk 2 above)
Comment (katia-energizer) [reply topics/setup.md:15]: Maybe rephrase. 
> Make sure that you have a [compatible Kotlin plugin] installed.

I would remove a specific Kotlin version here.

File: topics/setup.md (hunk 3 above)
Comment (katia-energizer) [reply topics/setup.md:26]: Agree.

File: topics/setup.md (hunk 1 above)
Comment (katia-energizer) [reply topics/setup.md:10]: Could you ask the question about AS version in the channel? 
From one side, it's not good to recommend only unreleased versions. However, we did this for the first version of the KMM plugin. 
For the other side, if we support only Beta versions, let's say this. I would avoid referring to Canary. Beta is OK.

As to Kotlin plugin version, see my comment below.

As to Xcode, I checked with the team some time ago and they said that 11.3 is OK. We don't have strict correlation.

File: topics/create-first-app.md [hunk 4]
```diff
@@ -154,4 +154,4 @@ Once you've played with your first KMM application, you can:
 
 * [Understand the KMM project structure](discover-kmm-project.md)
 * [Complete a hands-on tutorial on networking and data storage](hands-on-networking-data-storage.md)
-* [Integrate KMM in an existing application](integrate-in-existing-app.md)
\ No newline at end of file
+* [Make your Android application work on iOS – tutorial](integrate-in-existing-app.md)
```
Comment (katia-energizer) [inline topics/create-first-app.md:157]:
```suggestion
* [Complete a tutorial on making your Android application work on iOS](integrate-in-existing-app.md)
```
Let's keep it consistent with the previous item.

File: topics/create-first-app.md [hunk 5]
```diff
@@ -6,7 +6,7 @@ Here you will learn how to create and run your first KMM application.
 1. [Set up your environment for KMM development](setup.md) by installing the necessary tools on a suitable operating system.  
     
     > To work with shared code or Android-specific code, use any computer with an operating system supported by [Android Studio](https://developer.android.com/studio).
```
Comment (katia-energizer) [inline topics/create-first-app.md:8]: I've read this warning and it's not correct in this context. You need a mac with macOS for this tutorial, so there is no way.
I would use the text from the new tutorial:
> You will need a Mac with macOS to complete certain steps in this tutorial, which include writing iOS-specific code and running an iOS application.
> These steps cannot be performed on other operating systems, such as Microsoft Windows. This is due to an Apple requirement.

Also copyeditors are now removing an article before macOS. Please remove it in other places as well.

**Review by katia-energizer — CHANGES_REQUESTED** [review]
Overall looks good. Thanks for updating. Please check my comments.

**Review by katia-energizer — APPROVED** [review]
LGTM 👍
